store/models.py

```diff
@@ -2,12 +2,16 @@
 from django.contrib.auth.models import User
 from django.core.validators import MaxValueValidator, MinValueValidator
 
+from store.store_utils import StoreUtils
+
 
 class Store(models.Model):
     name = models.CharField(max_length=100)
     owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    category = models.CharField(max_length=30)
```

Comment:
Optional: Consider providing choices here. This allows you to better manage the types of categories in case you want to add additional logic based on category later.

```
CATEGORY_CHOICES = [
    ('RETAIL', 'Retail'),
    ('FOOD', 'Food'),
    ('SERVICE', 'Service'),
    # Add other categories as needed
]
category = models.CharField(max_length=30, choices=CATEGORY_CHOICES)
```

Reply:
Thank you for the suggestion. In a future version, this would be a nice feature.